src/components/show-hide-password-field/show-hide-password-field.js

```diff
@@ -0,0 +1,76 @@
+import React, {
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+} from 'react-intl';
+
+import {
+  Button,
+  TextField,
+} from '@folio/stripes/components';
+
+import css from './show-hide-password-field.css';
+
+const propTypes = {
+  field: PropTypes.node.isRequired,
+  hideButtonLabel: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
+  isHiddenBeDefault: PropTypes.bool,
+  showButton: PropTypes.bool,
+  showButtonLabel: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
+};
+
+const defaultProps = {
+  isHiddenBeDefault: true,
+  showButton: true,
+  showButtonLabel: <FormattedMessage id="ui-eholdings.showHidePasswordField.show" />,
+  hideButtonLabel: <FormattedMessage id="ui-eholdings.showHidePasswordField.hide" />,
+};
+
+const ShowHidePasswordField = ({
+  field: Field,
```

Comment:
Why do we need this prop? in my understanding that we use only one form - `react-final-form` and we can not just only import `Field` to this component?